### DISCUSSION TITLE
Aliases categories/groups

### DISCUSSION THREAD

**Discussion by juanitoddd** [ohmyzsh/ohmyzsh#10828] [Help] [answered]
I have used the **aliases** plugin to use the `acs` command.
I see there is some predefined groups/categories like
`[cd]`
        `1 = cd -1`
	`2 = cd -2`
	`...`

Is it posible to create a custom plugin `custom.zsh` an set a custom group/category
`[custom_group]`
         `foo=bar`

Thanks in advance

**Comment by mcornella** [comment, marked as answer]
The aliases plugin automatically groups aliases by the command, or else it puts them in the `_default` group if there are no other aliases for the same command. It's not made to customize the output, but a hacky way to do that is to define an alias that does nothing (`alias custom_group=''`) and use that as the first part of your aliases:

```sh
alias juanitoddd=''
alias alias1='juanitoddd ls -l'
alias alias2='juanitoddd cd /root'
alias alias3='juanitoddd pwd'
```

**Reply by juanitoddd** [reply]
Thanks for the reply and the explanation!

**Reply by Carlos-Marquez-StreamSets** [reply]
Super useful explanation, thank you!!
